packages/util/src/helpers.ts

```diff
@@ -1,5 +1,21 @@
 import { isHexString } from 'ethjs-util'
 
+/**
+ * Checks if a number string is a hex string, if not, converts and returns it 
```

Comment:
suggestion, this feels more clear

```suggestion
 * Returns a 0x-prefixed hex number string from a hex string or string integer.
```